dcsazure_AzureSQL_to_AzureSQL_prof_pl/dcsazure_AzureSQL_to_AzureSQL_prof_pl.json

```diff
@@ -0,0 +1,609 @@
+                      "type": "DataFlowReference",
+                      "parameters": {
+                        "runId": {
+                          "value": "'@{pipeline().RunId}'",
+                          "type": "Expression"
+                        },
+                        "DF_SOURCE_SCHEMA": {
+                          "value": "'@{item().specified_schema}'",
+                          "type": "Expression"
+                        },
+                        "DF_SOURCE_TABLE": {
+                          "value": "'@{item().identified_table}'",
+                          "type": "Expression"
+                        },
+                        "DF_NUM_ROWS_TO_PROFILE": "1000",
+                        "DF_METADATA_SCHEMA": {
+                          "value": "'@{variables('METADATA_SCHEMA')}'",
+                          "type": "Expression"
+                        },
+                        "DF_METADATA_RULESET_TABLE": {
+                          "value": "'@{variables('METADATA_RULESET_TABLE')}'",
+                          "type": "Expression"
+                        },
+                        "DF_SOURCE_DATABASE": {
+                          "value": "'@{item().specified_database}'",
+                          "type": "Expression"
+                        },
+                        "DF_SOURCE_DATASET": {
+                          "value": "'@{variables('DATASET')}'",
+                          "type": "Expression"
+                        }
+                      },
+                      "datasetParameters": {
+                        "Source1MillRowDataSampling": {},
+                        "MetadataStoreRead": {},
+                        "WriteToMetadataStore": {}
+                      }
+                    },
+                    "staging": {},
+                    "compute": {
+                      "coreCount": 8,
+                      "computeType": "General"
+                    },
+                    "traceLevel": "Fine"
+                  }
+                }
+              ]
+            }
+          }
+        ],
+        "policy": {
+          "elapsedTimeMetric": {}
+        },
+        "parameters": {
+          "P_SOURCE_DATABASE": {
+            "type": "string",
+            "defaultValue": "shared-azure-sql-server"
+          },
+          "P_SOURCE_SCHEMA": {
+            "type": "string",
+            "defaultValue": "test_sql_data"
+          }
+        },
+        "variables": {
+          "METADATA_SCHEMA": {
+            "type": "String",
+            "defaultValue": "dbo"
+          },
+          "METADATA_RULESET_TABLE": {
+            "type": "String",
+            "defaultValue": "discovered_ruleset"
+          },
+          "DATASET": {
+            "type": "String",
+            "defaultValue": "AZURESQL"
+          }
+        },
+        "folder": {
+          "name": "dcsazure_AzureSQL_to_AzureSQL_prof_pl"
```

Comment:
I've checked folder names in other pipelines and found that they kept the names as `dcsazure_AzureSQL_to_AzureSQL` without any prefix like `_prof_pl` or `_ds` (in case of datasets). I think we should follow the same convention here. We are anyway adding the suffixes like `_prof_pl` and `_ds` in the pipeline name and dataset name, respectively.

Reply:
I'll correct that. I think I also made changes to the folder while updating the dataset and pipeline name.